src/main/resources/routes

```diff
@@ -3,14 +3,16 @@
 # ~~~~
 
 # Map static resources from the /public folder to the /assets URL path
-GET         /assets/*file                                controllers.Assets.at(path="/public", file)
+GET         /assets/*file                                             controllers.Assets.at(path="/public", file)
 
-GET         /                                            controllers.StartPageController.startPage
-GET         /opportunities                               controllers.OpportunityController.showOpportunities
-GET         /opportunities/guidance/:opportunityId                      controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
-GET         /opportunity/:opportunityId                  controllers.OpportunityController.showOpportunity(opportunityId:OpportunityId, section:Option[Int])
+GET         /                                                         controllers.StartPageController.startPage
+GET         /opportunities                                            controllers.OpportunityController.showOpportunities
+GET         /opportunities/guidance/:opportunityId                    controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
+GET         /opportunity/:opportunityId                               controllers.OpportunityController.showOpportunity(opportunityId:OpportunityId, section:Option[Int])
 
-GET         /application_form/:id/apply                  controllers.ApplicationController.showOrCreateForForm(id: ApplicationFormId)
+GET         /manage/opportunity/new                                   controllers.OpportunityController.createOpportunity
```

Comment:
That should be PUT and also I'd rather have separate URLs for genuine "new"  & "re-use"